<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>976</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="frame_menu" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>151</width>
      <height>591</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color:qlineargradient(spread:pad, x1:1, y1:1, x2:0, y2:0, stop:0 rgba(83, 170, 189, 255), stop:1 rgba(255, 255, 255, 255));</string>
    </property>
    <widget class="QPushButton" name="btn_run">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>151</width>
       <height>71</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Run&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Run</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_algorithms">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>151</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>Algorithms</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_maps">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>140</y>
       <width>151</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>Maps</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_configuration">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>210</y>
       <width>151</width>
       <height>71</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Configuration</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layout_map" native="true">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>0</y>
      <width>811</width>
      <height>431</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(202, 202, 202)</string>
    </property>
    <layout class="QGridLayout" name="gridLayout"/>
   </widget>
   <widget class="QWidget" name="layout_algorithms" native="true">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>0</y>
      <width>811</width>
      <height>431</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(202, 202, 202)</string>
    </property>
   </widget>
   <widget class="QWidget" name="layout_run" native="true">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>0</y>
      <width>811</width>
      <height>431</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(202, 202, 202)</string>
    </property>
   </widget>
   <widget class="QWidget" name="layout_config" native="true">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>0</y>
      <width>811</width>
      <height>431</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(202, 202, 202)</string>
    </property>
   </widget>
   <zorder>frame_menu</zorder>
   <zorder>layout_map</zorder>
   <zorder>layout_run</zorder>
   <zorder>layout_algorithms</zorder>
   <zorder>layout_config</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>976</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDeUber">
    <property name="title">
     <string>Panel General</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuMi_cuenta">
    <property name="title">
     <string>Mi cuenta</string>
    </property>
   </widget>
   <addaction name="menuDeUber"/>
   <addaction name="menuMi_cuenta"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
